app/viewmodels/checkAnswers/expenses/tailoring/AdvertisingOrMarketingSummary.scala

```diff
@@ -18,29 +18,27 @@ package viewmodels.checkAnswers.expenses.tailoring
 
 import controllers.journeys.expenses.tailoring.routes
 import models.CheckMode
-import models.common.{BusinessId, TaxYear}
+import models.common.{BusinessId, TaxYear, UserType}
 import models.database.UserAnswers
 import pages.expenses.tailoring.AdvertisingOrMarketingPage
 import play.api.i18n.Messages
-import play.twirl.api.HtmlFormat
-import uk.gov.hmrc.govukfrontend.views.viewmodels.content.HtmlContent
-import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist.SummaryListRow
+import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist.{Key, SummaryListRow, Value}
 import viewmodels.govuk.summarylist._
 import viewmodels.implicits._
 
 object AdvertisingOrMarketingSummary {
 
-  def row(answers: UserAnswers, taxYear: TaxYear, businessId: BusinessId)(implicit messages: Messages): Option[SummaryListRow] =
-    answers.get(AdvertisingOrMarketingPage).map { answer =>
-      val value = ValueViewModel(
-        HtmlContent(
-          HtmlFormat.escape(messages(s"advertisingOrMarketing.$answer"))
-        )
-      )
-
+  def row()(implicit messages: Messages, answers: UserAnswers, taxYear: TaxYear, businessId: BusinessId, userType: UserType): Option[SummaryListRow] =
+    answers.get(AdvertisingOrMarketingPage, Some(businessId)).map { answer =>
       SummaryListRowViewModel(
-        key = "advertisingOrMarketing.checkYourAnswersLabel",
-        value = value,
+        key = Key(
```

Comment:
this part is duplictated in a lot of places. Maybe we can extract it to a helper function just like in SummaryListCYA?

contentKey, contentValue could be a parameter

Reply:
Will add to tech debt, thanks